### PR TITLE
Fixed issue #17100: New question will be added to the wrong group

### PR DESCRIPTION
**Pull request by gabrieljenik**
Reassigning group after part of the screen s refreshed

### CODE REVIEW

File: assets/scripts/admin/questionEditor.js [hunk 1]
```diff
@@ -1514,7 +1514,9 @@ $(document).on('ready pjax:scriptcomplete', function () {
       });
       try {
         const [html, html2] = await Promise.all([generalSettingsPromise, advancedSettingsPromise]);
+        var currentGroup = $('#gid').children("option:selected").val();
```
Comment (olleharstedt) [inline assets/scripts/admin/questionEditor.js:1517]: Can you use let or const instead, please?

**Review by olleharstedt — CHANGES_REQUESTED** [review]

File: assets/scripts/admin/questionEditor.js (hunk 1 above)
Comment (gabrieljenik) [reply assets/scripts/admin/questionEditor.js]: Done

Comment (olleharstedt) [reply assets/scripts/admin/questionEditor.js]: Sorry, I should have said: Use const (the value is never changed...)

Comment (gabrieljenik) [reply assets/scripts/admin/questionEditor.js]: Well, I disagree with you on this one. From its conception is a variable, not a constant... I would keep with let... But if you insist, of course I will change it :)

Comment (olleharstedt) [reply assets/scripts/admin/questionEditor.js]: `const` should be used for variables that do not change. Compare with the `const` keyword in C++. Thanks, though.

Comment (olleharstedt) [reply assets/scripts/admin/questionEditor.js]: https://docs.microsoft.com/en-us/cpp/cpp/const-cpp?view=msvc-160

**Comment by olleharstedt** [comment]
Thanks!

**Pull request merged**